pkgs/desktops/plasma-5/kwin/default.nix

```diff
@@ -2,17 +2,19 @@
   mkDerivation, lib,
   extra-cmake-modules, kdoctools, fetchpatch,
 
-  libepoxy, lcms2, libICE, libSM, libcap, libdrm, libinput, libxkbcommon, mesa,
-  pipewire, udev, wayland, xcb-util-cursor, xwayland,
+  libepoxy, lcms2, libICE, libSM, libcap, libdrm, libinput, libxkbcommon,
+  libxcvt, mesa, pipewire, plasma-wayland-protocols, udev, wayland,
+  wayland-protocols, xcb-util-cursor, xwayland,
```

Comment:
Please don't reflow this. It makes the diff unredable.

Reply:
moving all additions to a separate line + keep current formatting